### DISCUSSION TITLE
Localize content field

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16418] [Q&A] [answered]
Is it possible to localize a content field?

**Comment by Piedone** [comment, marked as answer]
If you mean, whether you can provide multiple versions of a content field's content for different cultures, without localizing the whole content item, then not (without coding). The recommendation is to localize the whole content item, including all fields, though.

**Comment by hishamco** [comment]
Localizing fields are not supported yet, we supported content item localization, which @Piedone refers to
